x/concentrated-liquidity/lp_test.go

```diff
@@ -1047,3 +1048,47 @@ func (s *KeeperTestSuite) TestUninitializePool() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestIsLockMature() {
+	type sendTest struct {
+		remainingLockDuration time.Duration
+		lockIsMature          bool
+	}
+	tests := map[string]sendTest{
+		"unlocked": {
+			remainingLockDuration: 0,
+			lockIsMature:          true,
+		},
+		"locked": {
+			remainingLockDuration: 1 * time.Hour,
+			lockIsMature:          false,
+		},
+	}
```

Comment:
Added on more test case here https://github.com/osmosis-labs/osmosis/pull/4854/commits/dfb7e738a9cbb900145ab0cdb3a1e2360495a8eb

This tests three of four paths. The fourth path is actually impossible to reach right now, since IsLockMature calls GetLockByID, and GetLockByID can only return one error. One of the branches allows for this error, the other branch returns an error if the error is not this one error.